### PR TITLE
create the connection and reddit.sql file and build.js file relates #4

### LINKED ISSUE #4: [backend] setup db schema
_TASK_
- setup the sql file 
- create the connection file 
- create build file 
- setup the connection

### PR DESCRIPTION
**Pull request by mo7amedehab97**
- create the connection.js file
- create  reddit.sql file 
- create  build.js file
 relates #4

### CODE REVIEW

File: src/db/config/connection.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require('env2')('.env');
+const { Pool } = require('pg');
+
+const { NODE_ENV } = process.env;
+let connectionString = '';
+let ssl = false;
+
+switch (NODE_ENV) {
+  case 'production':
+    connectionString = process.env.DATABASE_URL;
```
Comment (LinaYahya) [inline src/db/config/connection.js]: why not destructuring in line 4

File: src/db/config/reddit.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+);
+
+CREATE TABLE comments (
+    id SERIAL PRIMARY KEY,
+    user_id INT,
+    post_id INT,
+    comment TEXT NOT NULL,
+     user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+    post_id INTEGER REFERENCES posts(id) ON DELETE CASCADE,
+    created_at TIMESTAMP NOT NULL
```
Comment (LinaYahya) [inline src/db/config/reddit.sql]: add a default value as a date.now, so no need to insert value in insert query

**Review by LinaYahya — COMMENTED** [review]
Well Done Mohammed

File: src/db/config/reddit.sql (hunk 2 above)
Comment (mo7amedehab97) [reply src/db/config/reddit.sql]: done

File: src/db/config/connection.js (hunk 1 above)
Comment (mo7amedehab97) [reply src/db/config/connection.js]: done

**Pull request merged**